packages/integrations/sitemap/src/utils/is-route-prerendered.ts

```diff
@@ -0,0 +1,8 @@
+const REGEX_PRERENDER = /const prerender = true/g;
+import { readFile } from 'fs/promises';
+
+export async function isRoutePrerendered(filePath: string) {
+	const contents = await readFile(filePath, 'utf-8');
+
+	return REGEX_PRERENDER.test(contents);
+}
```

Comment:
Same comment about logic duplication. This info is handled in `astro` and surfaced in the `routes` map.